# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/referrer-policy/generic/inheritance/iframe-inheritance-about-blank.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/referrer-policy/generic/inheritance/iframe-inheritance-about-blank.html.ini
--- a/testing/web-platform/meta/referrer-policy/generic/inheritance/iframe-inheritance-about-blank.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/referrer-policy/generic/inheritance/iframe-inheritance-about-blank.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,10 +1,8 @@
 [iframe-inheritance-about-blank.html]
   expected:
-    if (os == "win") and debug and (processor == "x86_64") and webrender and not swgl: ["OK", "TIMEOUT"]
-    if (os == "linux") and webrender and not debug and not fission: ["OK", "TIMEOUT"]
-    if (os == "android") and debug and not webrender: ["OK", "TIMEOUT"]
-    if (os == "android") and debug and webrender: ["OK", "TIMEOUT"]
-    if (os == "android") and not debug and webrender: ["OK", "TIMEOUT"]
+    if (os == "android") and not webrender and debug: ["OK", "TIMEOUT"]
+    if (os == "android") and not webrender and not debug: ["OK", "TIMEOUT"]
+    if (os == "android") and webrender and debug: ["OK", "TIMEOUT"]
     if os == "mac": TIMEOUT
     [TIMEOUT, OK]
   [The value of document.referrer in an about:blank iframe is the outer document's full URL, regardless of referrer policy]
@@ -15,11 +13,9 @@
 
   [A subresource fetched from an about:blank iframe is fetched with no 'Referer' header]
     expected:
-      if (os == "win") and debug and (processor == "x86_64") and webrender and not swgl: ["PASS", "NOTRUN"]
-      if (os == "linux") and webrender and not debug and not fission: ["PASS", "NOTRUN"]
-      if (os == "android") and debug and webrender: ["PASS", "NOTRUN"]
-      if (os == "android") and debug and not webrender: ["PASS", "NOTRUN"]
-      if (os == "android") and not debug and webrender: ["PASS", "NOTRUN"]
+      if (os == "android") and webrender and debug: ["PASS", "NOTRUN"]
+      if (os == "android") and not webrender and debug: ["PASS", "NOTRUN"]
+      if (os == "android") and not webrender and not debug: ["PASS", "NOTRUN"]
       if os == "mac": NOTRUN
       [NOTRUN, PASS]
 